_documentation/05-User-Manual.md

```diff
@@ -201,3 +201,125 @@ Apart from configuring printing defaults for the printer, these are the other pr
 3. **Print Test Page**
 
     This option helps in testing printer functionality, by issuing a print command for a sample test page. 
+
+## Working with Command-Line
+
+Before start working with your device, you must know about the capabilities of your printer/scanner by viewing the set of sub-commands and options supported by your printer/scanner. For retrieving this information, you need to pass the `--help` argument:
+
+    <application-name> --help
+
+Although, each application may have a different usage-callback function, the default list of sub-commands and options available with all applications is shown below.
```

Comment:
If this is a user manual you should not mention the "usage-callback" function as this is an internal functionality of the Printer Application which only a developer sees, not a user. Simply write that some Printer Applications can have additional sub-commands.

Reply:
Thanks, Changed the content.